test/kokoro/xds.sh

```diff
@@ -28,7 +28,7 @@ GRPC_GO_LOG_VERBOSITY_LEVEL=99 GRPC_GO_LOG_SEVERITY_LEVEL=info \
     --gcp_suffix=$(date '+%s') \
     --verbose \
     --client_cmd="grpc-go/interop/xds/client/client \
-      --server=xds-experimental:///{server_uri} \
+      --server=xds:///{server_uri} \
```

Comment:
@ericgribkoff Is this the only change for the interop tests to use scheme `xds` instead of `xds-experimental`?

Reply:
Yes